### DISCUSSION TITLE
为啥还不发布新版本

### DISCUSSION THREAD

**Discussion by Tpfdd** [ywmoyue/biliuwp-lite#915] [General]
### 问题描述

一个月了，bug修了挺多怎么还不更新呀

### 描述解决方案

催更

### 代码贡献

否

**Comment by ywmoyue** [comment]
本来打算下个版本做的一些内容还没做完

**Comment by ywmoyue** [comment]
https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.7.7 已发布

**Reply by Tpfdd** [reply]
感谢，辛苦了
